NomaiGrandPrix/SpawnPoints.tsv

```diff
@@ -45,109 +59,95 @@ Spawn_DreamZone_3_SecretGallery	DreamZone 3 Secret Gallery	DreamZone	TRUE	FALSE
 Spawn_DreamZone_4_Bridge	DreamZone 4 Bridge	DreamZone	TRUE	FALSE	FALSE	FALSE	
 Spawn_DreamZone_4_Dock	DreamZone 4 Dock	DreamZone	TRUE	FALSE	FALSE	FALSE	
 Spawn_DreamZone_4_Elevator	DreamZone 4 Elevator	DreamZone	TRUE	FALSE	FALSE	FALSE	
-SPAWN_EscapePod	Escape Pod 1 (Brittle Hollow)	BrittleHollow	FALSE	TRUE	TRUE	FALSE	
-Spawn_EscapePod_Exterior	Escape Pod 2 (Ember Twin) Exterior	EmberTwin	FALSE	TRUE	TRUE	FALSE	
-Spawn_EscapePod_Interior	Escape Pod 2 (Ember Twin) Interior	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_SecretLibrary_1	Secret Library 1	DreamZone	FALSE	FALSE	FALSE	FALSE	Stranger
+Spawn_SecretLibrary_2	Secret Library 2	DreamZone	FALSE	FALSE	FALSE	FALSE	Stranger
+Spawn_SecretLibrary_3	Secret Library 3	DreamZone	FALSE	FALSE	FALSE	FALSE	Stranger
+Spawn_Underground_CodeTotems	Underground Code Totems	DreamZone	TRUE	FALSE	FALSE	FALSE	Stranger (Is this dreamzone? It has to be right?)
+Spawn_Underground_OOB	Underground OOB	DreamZone	TRUE	FALSE	FALSE	FALSE	Stranger
+Spawn_Underground_PrisonCell	Underground Prison Cell	DreamZone	TRUE	FALSE	FALSE	FALSE	Stranger
+Spawn_Underground_PrisonCell_Lower	Underground Prison Cell Lower	DreamZone	TRUE	FALSE	FALSE	FALSE	Stranger
+Spawn_ChertsCamp	Chert's Camp	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_EscapePod_Exterior	Escape Pod 2 Exterior	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_EscapePod_Interior	Escape Pod 2 Interior	EmberTwin	FALSE	FALSE	FALSE	FALSE	
 Spawn_FossilOverlook	Fossil Overlook	EmberTwin	FALSE	TRUE	TRUE	FALSE	
 Spawn_FossilSite	Fossil Site	EmberTwin	FALSE	TRUE	TRUE	FALSE	
```

Comment:
Maybe just "Fossil" or "Anglerfish Fossil"? (If the latter, probably add "Anglerfish" to it too)

Reply:
Sure, easy